client/js/components/day/staff-group.js

```diff
@@ -10,29 +11,57 @@ var StaffGroup = React.createClass({
   getDefaultProps: function(){
     return {
       approved: [],
-      needed: '',
-      name: ''
+      needed: Infinity,
+      name: '',
+      gigID: Infinity
     };
   },
 
   render: function(){
-    var approved = _.map(this.props.approved, function(human, idx){
-      return <Employee rating={human.rating} name={human.name} key={idx} />
+    var props = this.props;
+    var health = props.approved.length / props.needed;
+
+    var approved = _.map(props.approved, function(human, idx){
+      return <Employee
+              rating={human.rating}
+              name={human.name}
+              key={idx}
+              gigID={props.gigID}
+              employeeID={human.employeeID}
+              group={props.name} />
     });
 
-    var numApproved = this.props.approved.length;
-    var needed = this.props.needed;
-    var name = this.props.name;
+    approved = props.connectDropTarget(
+      <div style={{backgroundColor:'aquamarine'}}>
```

Comment:
Is this a temporary inline styling? Would love to have a css class for authenticated/non-authenticated instead incase we want to change the design for different companies down the line